### PR TITLE
Changed everything to src folder and improved routing

### LINKED ISSUE #5: Add all controllers, utitilies etc folders under src folder
In the server root, we should only have index.js and propably kontena.yaml and dockerfile.

### PR DESCRIPTION
**Pull request by paulisiirama**
Resolves #5

### CODE REVIEW

**Comment by harjuto** [comment]
g0g0 :+1:

File: server.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ db.once('open', function (callback) {
   console.log('Connected to database')
 });
 
-require('./utilities/db');
+require('./src/utilities/db');
```
Comment (harjuto) [inline server.js]: This should not be needed.

**Comment by harjuto** [comment]
MERGE PLOX

File: server.js (hunk 1 above)
Comment (paulisiirama) [inline server.js]: It is since db.js is not added to src/utilities/index.js. I'll consider doing this whole test data thing in some other way.

**Pull request merged**